rpm_spec/manageiq.spec.in

```diff
@@ -25,22 +27,24 @@ Summary:  %{product_summary}
 License:  Apache-2.0
 URL:      %{product_url}
 
-Source0:  %{name}-appliance-%{version}.tar.gz
-Source1:  %{name}-core-%{version}.tar.gz
-Source2:  %{name}-gemset-%{version}.tar.gz
-Source3:  %{name}-manifest-%{version}.tar.gz
+Source0:  %{name}-ansible-venv-%{version}.tar.gz
+Source1:  %{name}-appliance-%{version}.tar.gz
+Source2:  %{name}-core-%{version}.tar.gz
+Source3:  %{name}-gemset-%{version}.tar.gz
+Source4:  %{name}-manifest-%{version}.tar.gz
 
 %description
 %{product_summary}
 
 %prep
 umask 0002
-%setup -q -n %{appliance_builddir}
```

Comment:
I don't know why this one was different than the rest, but I made it match - if you know why I can undo it.

Reply:
I don't know why.